.yarnrc.yml

```diff
@@ -1,2 +1,9 @@
-yarnPath: .yarn/releases/yarn-3.1.0.cjs
+nmHoistingLimits: none
+
 nodeLinker: node-modules
+
+plugins:
+  - path: .yarn/plugins/@ojkelly/plugin-build.cjs
```

Comment:
What is this piece for?

Reply:
I added one yarn plugin to build and bundle ..... but bundle not working as expected but build is good.